.env.example

```
@@ -807,70 +807,154 @@ VOICE_LLM_MODEL=gpt-4o-automatic
-# with Google STT, set GOOGLE_APPLICATION_CREDENTIALS instead.
+# Fish Audio TTS (S2 Pro — 80% cheaper than ElevenLabs, 15s voice cloning)
+FISH_AUDIO_API_KEY=your-fish-audio-api-key
+# FISH_AUDIO_VOICE_ID=your-fish-audio-voice-id
+
+# =============================================================================
+# NEW VIDEO PROVIDERS
+# =============================================================================
+
+# Kling 3.0 (4K/60fps via PiAPI proxy)
+KLING_API_KEY=your-kling-api-key
+KLING_API_BASE=https://api.piapi.ai/api/kling/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KLING_API_BASE key should go before the KLING_API_KEY key

(UnorderedKey)